src/package_json.rs

```diff
@@ -213,19 +192,6 @@ impl PackageJson {
         Some(value)
     }
 
-    /// Raw serde json value of `package.json`.
-    ///
-    /// This is currently used in Rspack for:
-    /// * getting the `sideEffects` field
-    /// * query in <https://www.rspack.dev/config/module.html#ruledescriptiondata> - search on GitHub indicates query on the `type` field.
-    ///
-    /// To reduce overall memory consumption, large fields that useless for pragmatic use are removed.
-    /// They are: `description`, `keywords`, `scripts`,
-    /// `dependencies` and `devDependencies`, `peerDependencies`, `optionalDependencies`.
-    pub fn raw_json(&self) -> &Arc<serde_json::Value> {
```

Comment:
!!! I need this API for Rspack :-(

See comments

    /// This is currently used in Rspack for:
    /// * getting the `sideEffects` field
    /// * query in <https://www.rspack.dev/config/module.html#ruledescriptiondata> - search on GitHub indicates query on the `type` field.

Reply:
Although it's tempting to provide another set of APIs, the changes within Rspack is not something I want to tackle at the moment ...

Reply:
@Boshen Do you know which fields rspack needs? I can probably have this return the `data.other_fields` map.

Reply:
"query in description data" means all of it ... I need that original serde json :-(